config/routes.rb

```diff
@@ -1,3 +1,21 @@
+#Draw routes
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  #Set root path
+  root 'patients#index'
+
+  #Set admin path
+  get 'admin', to: 'access#menu'
+
+  #Set paths to handle admin access
+  get 'access/menu'
+  get 'access/login'
+  post 'access/attempt_login'
+  get 'access/logout'
+
+  resources :patients do  
```

Comment:
haven't seen this kind of route before, what is it doing?

Reply:
resources creates a route related to every CRUD action besides Delete
[https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions](url)